pkg/cadvisor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM google/cadvisor:v0.28.0@sha256:09c8d73c9b799d30777763b7029cfd8624b8a1bd33af652ec3b51a6b827d492a
```

Comment:
This is x86_64 only, so `build.yml` below muct restrict the arch to that. Alternatively, it could build from source and do so for x86_64 and arm64

Reply:
building from source is not done even in the upstream Dockerfile. What about "go get github.com/google/cadvisor" ?

Reply:
the problem in this case, would be repeatability. i'm not familiar enough with go tools to install a specific commit

Reply:
I've looked it up. I suppose go get -d ... git branch ... make

Reply:
@mmariani you may want to model it after https://github.com/linuxkit/linuxkit/tree/master/pkg/vpnkit-expose-port

Reply:
ok, will do. thanks

Reply:
Would the current version be ok? I did a static binary, in case the shell script is not required anymore in the future, although it seems to use dlopen() for some feature (nvidia monitoring, etc)

Reply:
I'd be fine with the current version but then it also needs to have:
```
arches:
  - amd64
```
in the `build.yml`

Reply:
Sorry, I don't understand. It builds from sources and should work on arm64 too, I only used the linuxkit/alpine image. I see others use pretty much the same dockerfile for both: https://github.com/kontena/cadvisor-docker , am I missing something?

Reply:
My apologies. The GH UI showed me the old diff and I didn't spot the refresh